app/controllers/api/comments_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Api::CommentsController < Api::ApplicationController
-  before_action :update_and_set_user
+  before_action :set_user
+  before_action :update_username_if_changed
```

Comment:
Po tych wszystkich zmianach widzę, że żeby kod był czysty po stronie backendu, to trzeba było jednak zrobić tak, żeby to apka mobilna wysyłała osobny call, gdy user zmienia nick (nie jako powtórzenie calla z logowaniem do debaty, tylko osobno).

Ale teraz już to zostawmy, nie ma co tego robić przed czwartkiem.